docs/source/user-guide/vision/vision_properties.rst

```diff
@@ -0,0 +1,188 @@
+===============
+Data Properties
+===============
+
+Some of the Deepchecks' checks (e.g. train-test drift) use "Properties". Properties are simple one-dimension values that
+are extracted either from the images, labels or predictions of the data.
+
+For example, one image property is **brightness**, and one label property is **bounding box area** (for detection
+tasks). Each property produces a single value per image or per individual label in the image.
+
+What Are Properties Used For?
+=============================
+
+The properties values' distribution is a characteristic of your data that might have hidden a relation to your
+model performance. They might come handy in a few situations:
+
+1. New unlabeled data: To approximate how your model will work on the new data, since you don't have labels
+   in order to do a performance check. If the new data's properties are different by a large margin from the
+   train's properties, then it might indicate the model will have harder time inferring on it.
+2. Low test performance: If your model is having low performance on your test data, a good step in debugging
+   it is to check the drift of different properties between the train and the test. If there are some
+   properties with a high drift it might help you pinpoint to a relevant causes of the model's low performance.
+3. Find leakage in the model: Sometimes the model training might be affected the properties we are not aware of,
+   and that aren't the core objective we are aiming to learn. For example, in a classification dataset of wolves
+   and dogs photographs, if only wolves are photographed in the snow, the brightness of the image may be used to
+   predict the label "wolf" easily. In this case, a model might not learn to discern wolf from dog by the animal's
+   characteristics, but by using the background color.
+4. Find weak segments: The properties can be used to segment the data and test for low performing segments.
+   If found, the weak segment might indicate a gap in the training data where the data quality is worse.
+
+Deepchecks' Default Properties
+==============================
+
+We divide the data into 3 parts: images, labels and predictions. Each one have his own properties defined.
+When running checks you can either create your own properties or relay on Deepchecks' default properties.
+
+The default image properties are:
+
+- Aspect Ratio
+- Area
+- Brightness
+- RMS (Root Mean Square) Contrast
+- Normalized Mean RGB channels: The normalized channels are calculated by first normalizing the image's pixels (meaning, each color is normalized
+  to its relevant intensity, by dividing the color intensity by the other colors). Then, the mean for each
+  image channel is calculated.
```

Comment:
Can be shorter:

Mean color intensity for each channel. The color intensity is normalized according to the other color channels per pixel. This is done in order to capture the relationships between channels and not just general intensity (brightness)